spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_common.rb

```diff
@@ -863,5 +863,107 @@ def expect_sync_cloud_tenants_with_tenants_is_queued
       expect(sync_cloud_tenant.method_name).to eq("sync_cloud_tenants_with_tenants")
       expect(sync_cloud_tenant.state).to eq(MiqQueue::STATE_READY)
     end
+
+    def assert_targeted_vm(vm_name, attributes)
```

Comment:
Right so what I do in AWS is a full VM refresh now, so I expand more targets out of the Vm, so the result is a Vm with all network connections(IPs) and volumes.